.babelrc

```diff
@@ -1,3 +1,10 @@
 {
-  "presets": ["es2015", "stage-0", "react"]
+  "presets": [
+    ["es2015", {"modules": false}],
```

Comment:
@Kikobeats it will work either way.  disables modules tell that es6 import will be handled by webpack and not transpiled by babel to commonjs